src/bucket/BucketOutputIterator.cpp

```diff
@@ -76,6 +70,18 @@ BucketOutputIterator::put(BucketEntry const& e)
     *mBuf = e;
 }
 
+bool
+BucketOutputIterator::enforceSorted(BucketEntry const& e)
```

Comment:
per our chat, I don't think we need to mess with out of order entries, ie we don't need this method `enforceSorted`